tests/bluetooth/tester/src/gap.c

```diff
@@ -1094,6 +1122,13 @@ uint8_t tester_init_gap(void)
 {
 	int err;
 
+	cb.pairing_accept = auth_pairing_accept;
+
+	err = bt_conn_auth_cb_register(&cb);
+	if (err && err != -EALREADY) {
+		return BTP_STATUS_FAILED;
+	}
```

Comment:
I think it would make sense to add this change to the other occurences of `bt_conn_auth_cb_register` in this function as well. Alternatively, move the CB register to a central place that ensure it is only called once.

Reply:
The other call is preceded by a call with a `NULL` argument to clear the registered callback.

Reply:
Not the one in line 805 :)

Reply:
It is, just ~30 lines above :)

Reply:
You are correct :) That would also be an alternative solution for this; either way works really.